pages/docs/cloud/repository-integration.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Grouparoo Cloud Repository Integration"
+date: "2021-11-10"
+pullQuote: "Simple integration steps for Grouparoo Cloud"
+---
+
+Grouparoo Cloud requires three things to function: A github action yaml file and two secrets to identify and authorize the code config repository.
+
+It is assumed you have an existing repository on github that was setup using our [Config UI](/docs/config).
+
+### GitHub Repository Secrets
+
+Grouparoo Cloud requires two secrets to be available in your repository: `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID`.
+
+If these values have not already been provided for you, you can get them from [Grouparoo Cloud](https://cloud.grouparoo.com/) in your Project's Integrations page.
+
+In your GitHub repository go to: Settings -> Secrets -> New repository secret.
+
+<Image
+  src="docs/cloud/action-secrets.png"
+  alt="GitHub Repository Secrets"
+  width={1127}
+  height={650}
+/>
+
+Add `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID` with the provided values.
+
+<sub>The Project ID should look similar to `prj_some-set-of-values`.</sub>
```

Comment:
Why is this sentence `<sub>`? Seems like the backticks also don't work here.

Reply:
To function more like a footnote. If the backticks are showing through I can remove them.